dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/Connector.java

```diff
@@ -50,6 +52,25 @@ default String getDescription() {
    */
   default void validate(ConnectorArguments arguments) throws IllegalStateException {}
 
+  default void validateDatesRange(ConnectorArguments arguments) {
```

Comment:
```suggestion
  default void validateDateRange(ConnectorArguments arguments) {
```

Let's use in all places `dateRange`. As it has be used here:
```
boolean useDefaultDateRangeToFetch = arguments.getStartDate() == null;
```

Reply:
ok.